int32[] ball_xyxy
int32[] goalpost_xyxy
int32[] robot_xyxy
int32[] L_mark_xyxy
int32[] T_mark_xyxy
int32[] X_mark_xyxy